staff/emma-cowdery/mailchimp/index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mailchimp</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+</head>
+
+<body>
+    <header class="header">
+        <p class="header__login">Log In</p>
+        <nav class="header__nav">
+            <img src="./images/main_logo.png" class="header__nav-logo">
+            <button class="header__nav-signup">Sign Up Free</button>
+            <div class="header__nav-options">
+                <p class="header__nav-options-curvatureleft">Pricing</p>
+                <p class="header__nav-options-others">Features</p>
+                <p class="header__nav-options-others">Resources</p>
+                <p class="header__nav-options-others">Support</p>
+                <p class="header__nav-options-others">Blog</p>
```

Comment:
Since it is a Menu and it should redirect to Features page, resources page etc etc... I would use a `<a>` and in source put `#`. Anyway comment it with the assistant :)

Reply:
suggestion implemented